lib/duffel_api/resources/airline.rb

```diff
@@ -13,12 +13,20 @@ class Airline < BaseResource
       # @return [String]
       attr_reader :name
 
+      # @return [String]
```

Comment:
We were checking the the docs, actually it can be `nil`. (thanks @nlopes )

```suggestion
      # @return [String, nil]
```